2022-01-15 15:33:48,716 INFO    Thread-18 :23577 [wandb_setup.py:_flush():71] setting env: {'project': 'Amazon-Reviews', 'entity': 'ewebid', 'root_dir': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models', 'run_id': 'b8mnj89g', 'sweep_param_path': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models/wandb/sweep-j2xo5oqt/config-b8mnj89g.yaml', 'sweep_id': 'j2xo5oqt'}
2022-01-15 15:33:48,716 INFO    Thread-18 :23577 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-15 15:33:48,717 INFO    Thread-18 :23577 [wandb_init.py:_log_setup():371] Logging user logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models/wandb/run-20220115_153348-b8mnj89g/logs/debug.log
2022-01-15 15:33:48,717 INFO    Thread-18 :23577 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/src/models/wandb/run-20220115_153348-b8mnj89g/logs/debug-internal.log
2022-01-15 15:33:48,717 INFO    Thread-18 :23577 [wandb_init.py:init():404] calling init triggers
2022-01-15 15:33:48,717 INFO    Thread-18 :23577 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 200, 'drop_out': 0.15, 'epochs': 5, 'lr': 0.01, 'optimizer': 'sgd'}
config: {}
2022-01-15 15:33:48,717 INFO    Thread-18 :23577 [wandb_init.py:init():460] starting backend
2022-01-15 15:33:48,718 INFO    Thread-18 :23577 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-15 15:33:48,722 INFO    Thread-18 :23577 [backend.py:ensure_launched():216] starting backend process...
2022-01-15 15:33:48,728 INFO    Thread-18 :23577 [backend.py:ensure_launched():221] started backend process with pid: 23701
2022-01-15 15:33:48,733 INFO    Thread-18 :23577 [wandb_init.py:init():469] backend started and connected
2022-01-15 15:33:48,735 INFO    Thread-18 :23577 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 200, 'drop_out': 0.15, 'epochs': 5, 'lr': 0.01, 'optimizer': 'sgd'}
2022-01-15 15:33:48,740 INFO    Thread-18 :23577 [wandb_init.py:init():533] updated telemetry
2022-01-15 15:33:48,779 INFO    Thread-18 :23577 [wandb_init.py:init():563] communicating current version
2022-01-15 15:33:51,856 INFO    Thread-18 :23577 [wandb_init.py:init():568] got version response 
2022-01-15 15:33:51,856 INFO    Thread-18 :23577 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-15 15:33:52,172 INFO    Thread-18 :23577 [wandb_init.py:init():606] starting run threads in backend
2022-01-15 15:33:57,189 INFO    Thread-18 :23577 [wandb_run.py:_console_start():1810] atexit reg
2022-01-15 15:33:57,190 INFO    Thread-18 :23577 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-15 15:33:57,190 INFO    Thread-18 :23577 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-15 15:33:57,191 ERROR   Thread-18 :23577 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/opt/anaconda3/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-01-15 15:33:57,191 INFO    Thread-18 :23577 [wandb_init.py:init():633] run started, returning control to user process
2022-01-15 15:34:01,573 INFO    Thread-18 :23577 [wandb_run.py:finish():1457] finishing run ewebid/Amazon-Reviews/b8mnj89g
2022-01-15 15:34:01,574 INFO    Thread-18 :23577 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-15 15:34:01,577 INFO    Thread-18 :23577 [wandb_run.py:_restore():1752] restore
2022-01-15 15:34:01,590 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 757
}

2022-01-15 15:34:01,986 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 757
}

2022-01-15 15:34:02,236 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18055
}

2022-01-15 15:34:02,346 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18055
}

2022-01-15 15:34:02,452 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18055
}

2022-01-15 15:34:02,561 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18055
}

2022-01-15 15:34:02,674 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18055
}

2022-01-15 15:34:02,792 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18055
}

2022-01-15 15:34:02,906 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 757
  total_bytes: 18055
}

2022-01-15 15:34:03,016 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18055
  total_bytes: 18055
}

2022-01-15 15:34:03,124 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18055
  total_bytes: 18055
}

2022-01-15 15:34:03,235 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18055
  total_bytes: 18055
}

2022-01-15 15:34:03,342 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18055
  total_bytes: 18055
}

2022-01-15 15:34:03,445 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18055
  total_bytes: 18055
}

2022-01-15 15:34:03,554 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18055
  total_bytes: 18055
}

2022-01-15 15:34:04,499 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18055
  total_bytes: 18055
}

2022-01-15 15:34:05,476 INFO    Thread-18 :23577 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18055
  total_bytes: 18055
}
local_info {
}

2022-01-15 15:34:06,746 INFO    Thread-18 :23577 [wandb_run.py:_append_files():2180] logging synced files
